omniledger/service/service.go

```diff
@@ -421,13 +417,69 @@ func (s *Service) db() *skipchain.SkipBlockDB {
 	return s.skService().GetDB()
 }
 
+func (s *Service) loadConfig(scID skipchain.SkipBlockID) (*Config, error) {
+	coll := s.getCollection(scID)
+	// Find the genesis-darc ID.
+	val, contract, err := coll.GetValueContract(GenesisReferenceID.Slice())
+	if err != nil {
+		return nil, err
+	}
+	if string(contract) != ContractConfigID {
+		return nil, errors.New("did not get " + ContractConfigID)
+	}
+	if len(val) != 64 {
```

Comment:
Why is this not 32 bytes? If you put the darc-id in the val, then it's only 32 bytes. And the darc is by definition stored in the `[darcID:zeroNonce]`.